### ISSUE TITLE
Any good suggestions on debugging cloud functions

### ISSUE THREAD

**Issue by likeben** [graphcool/prisma#1662]
I mean how to test and debug my cloud functions. When I write a 'console.log(XXX)' in my code, I don't know where I can see the result. I try the 'graphcool-framework logs' command, but it only show the event object of each request. I also use the 'graphcool-framework invoke-local' command to execute the function, it doesn't work well either.

**Comment by jdheeter** [comment]
I spent many hours debugging graphcool-framework managed functions and had many problems and unexplained issues come up which I was unable to resolve. The feature has been dropped in the latest version, so I doubt that the team will improve the functionality. I ended up using the cli at serverless.com and the webhook functionality in graphcool to point at serverless api endpoints. It's a bit more complex to set up but probably more reliable and robust in the long term.
